l10n_ar_ux/__init__.py

```diff
@@ -41,3 +42,18 @@ def _inverse_l10n_latam_document_number(self):
                                       ' the POS number if there is not posted (or posted before) invoices'))
 
     AccountMove._inverse_l10n_latam_document_number = _inverse_l10n_latam_document_number
+
+    orginal_method_fiscal_position = AccountFiscalPosition._get_fiscal_position
+
+    #monkey patch
+    @api.model
+    def _get_fiscal_position(self, partner, delivery=None):
+        orginal_method_fiscal_position(self, partner, delivery=None)
+        company = self.env.company
+        if company.country_id.code == "AR":
+            self = self.with_context(
+            company_code='AR',
+            l10n_ar_afip_responsibility_type_id=partner.l10n_ar_afip_responsibility_type_id.id)
+        return super(AccountFiscalPosition, self)._get_fiscal_position(partner, delivery=delivery)
```

Comment:
we are calling bothm in  orginal_method_fiscal_position and also here with this super return? actually we need both?